core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -1021,6 +1092,17 @@ pub static mp_module_trezorui2: Module = obj_module! {
     ///     """Confirm byte sequence data."""
     Qstr::MP_QSTR_confirm_blob => obj_fn_kw!(0, new_confirm_blob).as_obj(),
 
+    /// def confirm_properties(
+    ///     *,
+    ///     title: str,
+    ///     items: Iterable[Tuple[str, str, bool]],
```

Comment:
...speaking of, this new type annotation disallows `None`s anyway?

Reply:
Good catch, fixed in 8a4c769cc2601aa8a3d01bd997a639cc6c5eaa51